stable/jenkins/templates/_helpers.tpl

```diff
@@ -103,11 +103,7 @@ jenkins:
       templates:
       - containers:
         - alwaysPullImage: {{ .Values.agent.alwaysPullImage }}
-          {{- if .Values.agent.args }}
-          args: "{{ .Values.agent.args }}"
-          {{- else }}
-          args: "^${computer.jnlpmac} ^${computer.name}"
-          {{- end }}
+          args: "{{ .Values.agent.args | replace "$" "^$"}}"
```

Comment:
nit: add a space before the the closing braces

Reply:
Thanks. Added it.